.ci/deps.sh

```diff
@@ -175,3 +175,10 @@ fi
 echo '' > dummy
 coala-ci --bears CheckstyleBear --files dummy --no-config --bear-dirs bears || true
 coala-ci --bears ScalaLintBear --files dummy --no-config --bear-dirs bears || true
+
+if [ ! -e ~/elm-format-0.18/elm-format ]; then
+  mkdir -p ~/elm-format-0.18
+  curl -fsSL -o elm-format.tgz https://github.com/avh4/elm-format/releases/download/0.5.2-alpha/elm-format-0.17-0.5.2-alpha-linux-x64.tgz
+  tar -xvzf elm-format.tgz -C ~/elm-format-0.18
+fi
+
```

Comment:
SpaceConsistencyBear had complained about it. :(, nvm removed the blank line. :)